scripts/gosec.sh

```diff
@@ -2,11 +2,32 @@
 
 set -o pipefail
 
+case "${1:-}" in
```

Comment:
if you want to parse it, you can better use real options with `getopt`. In case `CEPH_TAG` is not set, just do not pass `-tags` at all.

Requiring changing scripts on a Ceph update is not ideal.

Reply:
@nixpanic it looks like we do similar parsing in other scripts too. Considering that, may be atleast time time we will stick to this parsing. We can change all together later if required.

Reply:
Sorry, where are we parsing the Ceph version? I can not find it in other scripts.